### DISCUSSION TITLE
trusted_connection for MS SQL

### DISCUSSION THREAD

**Discussion by bloukanov** [sfu-db/connector-x#145] [Ideas]
Hi, has there been any discussion around adding this functionality? This would allow the Microsoft Integrated login rather than SQL Server login.

**Comment by wangxiaoying** [comment]
@bloukanov Yes, this functionality may benefit many mssql users. We will look into this.

**Comment by wangxiaoying** [comment]
Hi @bloukanov , trusted_connection is now supported for windows and will be included in our next release (0.2.3). It is also available now on alpha version, please also feel free to try by installing version `0.2.3-alpha.1`.

**Reply by bloukanov** [reply]
Apologies for the late response but we are indeed using it and it works great!

**Comment by bloukanov** [comment]
Fantastic, thanks for letting me know! We will try it out soon.
